packages/mobile/src/home/WalletHome.test.tsx

```diff
@@ -1,139 +1,133 @@
+      ...balances,
+      recipients: {
+        phoneRecipientCache: {},
+      },
+    })
+
+    jest.runAllTimers()
+
     expect(tree).toMatchSnapshot()
+    expect(tree.queryByTestId('HomeTokenBalance')).toBeFalsy()
+    expect(tree.queryByTestId('cashInBtn')).toBeFalsy()
+    expect(store.getActions()).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          "type": "SENTRY/INITIALIZE_SENTRY_USER_CONTEXT",
+        },
+        Object {
+          "action": null,
+          "alertType": "message",
+          "buttonMessage": null,
+          "dismissAfter": 5000,
+          "displayMethod": 0,
+          "message": "testnetAlert.1, {\\"testnet\\":\\"Alfajores\\"}",
+          "title": "testnetAlert.0, {\\"testnet\\":\\"Alfajores\\"}",
+          "type": "ALERT/SHOW",
+          "underlyingError": undefined,
+        },
+        Object {
+          "type": "HOME/REFRESH_BALANCES",
+        },
+        Object {
+          "doMatchmaking": false,
+          "type": "IDENTITY/IMPORT_CONTACTS",
+        },
+      ]
+    `)
   })
-  it('Renders when connected with backup complete', async () => {
-    const store = createMockStore()
-    const tree = render(
-      <Provider store={store}>
-        <WalletHome
-          refreshAllBalances={jest.fn()}
-          initializeSentryUserContext={jest.fn()}
-          setLoading={jest.fn()}
-          showMessage={jest.fn()}
-          loading={false}
-          recipientCache={{}}
-          numberVerified={true}
-          importContacts={jest.fn()}
-          balances={balances}
-          cashInButtonExpEnabled={false}
-          showTokensInHome={false}
-          {...getMockI18nProps()}
-        />
-      </Provider>
-    )
-    expect(tree).toMatchSnapshot()
+
+  it('Renders balances in home if feature flag is enabled', async () => {
+    const { getByTestId } = renderScreen({
+      ...balances,
+      app: {
+        multiTokenShowHomeBalances: true,
+      },
+    })
+
+    expect(getByTestId('HomeTokenBalance')).toBeTruthy()
   })
+
   it('Renders cash in bottom sheet when experiment flag is turned on and balances are zero', async () => {
-    const store = createMockStore()
-    const tree = render(
-      <Provider store={store}>
-        <WalletHome
-          refreshAllBalances={jest.fn()}
-          initializeSentryUserContext={jest.fn()}
-          setLoading={jest.fn()}
-          showMessage={jest.fn()}
-          loading={false}
-          recipientCache={{}}
-          numberVerified={true}
-          importContacts={jest.fn()}
-          balances={zeroBalances}
-          cashInButtonExpEnabled={true}
-          showTokensInHome={false}
-          {...getMockI18nProps()}
-        />
-      </Provider>
-    )
-    expect(tree).toMatchSnapshot()
+    const { getByTestId } = renderScreen({
+      ...zeroBalances,
+      app: {
+        cashInButtonExpEnabled: true,
+      },
+    })
+
+    expect(getByTestId('cashInBtn')).toBeTruthy()
```

Comment:
Love seeing more useful tests and not just a big snapshot test that nobody looks at ;)